components/Tabs/Tabs.js

```diff
@@ -69,4 +82,8 @@ class TabCard {
 - In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
 
 */
-let tabs = document.querySelectorAll();
\ No newline at end of file
+let tabs = document.querySelectorAll('.tab');
+
+tabs.forEach(element => {
+  new TabLink(element);
+})
```

Comment:
This is perfectly fine, but here is a way to write this in one line:

let tabs = document.querySelectorAll('.tab').forEach(tab => new TabLink(tab))